src/ddoc/highlight.d

```diff
@@ -133,6 +133,28 @@ $(D_CODE $(D_KEYWORD unittest) {
 	assert(r1 == e1, r1);
 }
 
+unittest {
+	auto s = `
+        ---------
+        asm pure nothrow @nogc @trusted
+        {
+            // the compiler does not check the attributes
+            ret;
+        }
+        ---------
+`;
+	auto e = `
+        $(D_CODE $(D_KEYWORD asm) $(D_KEYWORD pure) $(D_KEYWORD nothrow) @nogc @trusted
```

Comment:
Sorry, what I meant was "Is this first line of example code supposed to be indented further than the opening brace on the next line?".